tests/scripts/thread-cert/node.py

```diff
@@ -939,6 +939,10 @@ def srp_client_start(self, server_address, server_port):
         self.send_command(f'srp client start {server_address} {server_port}')
         self._expect_done()
 
+    def srp_client_autostart(self):
+        self.send_command('srp client autostart enable')
+        self._expect_done()
+
```

Comment:
I think we don't need this. We already have  `srp_client_enable_auto_start_mode()` which does the same.

Reply:
Thanks! Didn't notice that.